config.yaml

```diff
@@ -19,3 +19,20 @@ casa:
     url: "https://biztera.com/api/v1/"
     lookup_period_in_days: 30
     bot_email: 'bizbot@biztera.com'
+foxsec:
+    assignee:
+        - culucenk@mozilla.com
```

Comment:
Should the auto-assign got to as relevant FoxSec triager rather than Cag?

Reply:
Pretty sure I changed this in a more recent commit to `jvehent@mozilla.com`. I just used my address for testing first.